tests_e2e/scenarios/testsuites/agent_junit.py

```diff
@@ -28,7 +28,7 @@
 from lisa import schema  # pylint: disable=E0401
 from lisa.messages import (  # pylint: disable=E0401
     MessageBase,
-    TestResultMessage,
+    TestResultMessageBase,
```

Comment:
Using the Base class also captures SubTests. We don't have any of those currently, but decided to make the code more generic in case we use them.